### DISCUSSION TITLE
Error: Attempted import error: 'useRef' is not exported from 'react'

### DISCUSSION THREAD

**Discussion by TommySorensen** [adobe/react-spectrum#5686] [Q&A] [answered]
Am i the only one getting this error when using latest NextJs and latest react-aria?
```
Attempted import error: 'useRef' is not exported from 'react' (imported as '$6nfFC$useRef').
```

<img width="995" alt="image" src="https://github.com/adobe/react-spectrum/assets/4134473/31adf170-3b86-406a-884a-3b2adae97317">

**Comment by snowystinger** [comment]
Hey there, were you able to sort this out? Sorry, notification slipped through the cracks there.

**Reply by TommySorensen** [reply, marked as answer]
I think you can close it yes. Works fine in latest version 👍
